fleece/cli/run/run.py

```diff
@@ -156,11 +160,13 @@ def run(args):
 
     if args.environment:
         config = get_config(args.config)
-        account, role = get_account(config, args.environment)
+        account, role, username, apikey = get_account(config, args.environment)
     else:
         account = args.account
 
-    token, tenant = get_rackspace_token(args.username, args.apikey)
+    username = username or args.username
```

Comment:
It looks like this will override the `--username` and `--apikey` arguments from the command line if the file has `rs_username_var` or `rs_apikey_var` defined, which could be surprising.

Maybe printing a warning would be enough to mitigate this:
```
if username and args.username and username != args.username:`
    print('warning: --username argument is overridden by rs_username_var setting from YAML file.') 
if apikey and args.apikey and apikey != args.apikey:`
    print('warning: --apikey argument is overridden by rs_apikey_var setting from YAML file.') 
```

Reply:
Rethinking this, maybe it would be more clear if the CLI arguments took precedence. So you would need to not pass a username/apikey in the command-line if you want the custom env vars to be used.